lib/onigumo.ex

```diff
@@ -8,20 +8,21 @@ defmodule Onigumo do
     http_client = Application.get_env(:onigumo, :http_client)
     http_client.start()
 
-    download(http_client, @output_path)
+    Application.get_env(:onigumo, :input_path)
+    |> download_urls(http_client, @output_path)
```

Comment:
```suggestion
    |> download_urls_from_file(http_client, @output_path)
```

Reply:
Renamed in https://github.com/Glutexo/onigumo/pull/68/commits/8c7b6716af094a5415c999be9835ad2d72630bcc.